Source/WebKit/UIProcess/ApplicationStateTracker.mm

```diff
@@ -50,6 +51,28 @@ - (BOOL)_isHostedInAnotherProcess;
 
 namespace WebKit {
 
+static WeakHashSet<ApplicationStateTracker>& allApplicationStateTrackers()
+{
+    static NeverDestroyed<WeakHashSet<ApplicationStateTracker>> trackers;
+    return trackers;
+}
+
+static void updateApplicationBackgroundState()
+{
+    static bool isInBackground = false;
+    auto isAnyStateTrackerInBackground = []() -> bool {
```

Comment:
Or maybe here you mean to say "areAllStateTrackersInBackground"?

Reply:
Yes.

Reply:
FWIW, if can arrange to use anyOf() and a not operator, that's a little faster. This is not terribly important.

Reply:
Done.